housekeeping-vacuum-tool/src/main/java/com/hotels/housekeeping/tool/vacuum/conf/TunnelRouteValidator.java

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright (C) 2016-2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.housekeeping.tool.vacuum.conf;
+
+import static java.util.regex.Pattern.CASE_INSENSITIVE;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+import java.net.IDN;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class TunnelRouteValidator implements ConstraintValidator<TunnelRoute, CharSequence> {
```

Comment:
Ditto

Reply:
This class is still in Circus Train so I'm assuming its needed

Reply:
And this? https://github.com/HotelsDotCom/hcommon-ssh/blob/master/src/main/java/com/hotels/hcommon/ssh/validation/validator/TunnelRouteValidator.java

Reply:
You're right, I am going to create a PR against Circus Train that removes the reimplementation of these classes and have updated this PR